config/src/main/resources/config-repo/gateway-service.yml

```
@@ -160,3 +156,7 @@ springdoc:
         url: /api/users/v3/api-docs
       - name: venue-service
         url: /api/venues/v3/api-docs
+
+redis:
+  host: ${REDIS_HOST}
+  port: ${REDIS_PORT}
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)